<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountsWidgetBase</class>
 <widget class="QWidget" name="AccountsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Account</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="toolTip">
        <string>Add a new account</string>
       </property>
       <property name="text">
        <string>&amp;Add...</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Modify the selected account</string>
       </property>
       <property name="text">
        <string>&amp;Modify...</string>
       </property>
       <property name="icon">
        <iconset theme="edit-rename">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove the selected account</string>
       </property>
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnUp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="icon">
        <iconset theme="go-up">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDown">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="icon">
        <iconset theme="go-down">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="accountsTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Alias</string>
      </property>
      <property name="toolTip">
       <string>An alias for the account</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Micro-Blog</string>
      </property>
      <property name="toolTip">
       <string>The type of the account</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Enabled</string>
      </property>
      <property name="toolTip">
       <string>Whether the microblog is enabled or disabled</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Read Only</string>
      </property>
      <property name="toolTip">
       <string>Read Only account</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quick Post</string>
      </property>
      <property name="toolTip">
       <string>Show in Quick Post</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>accountsTable</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnRemove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
